code/controllers/subsystem/afk.dm

```diff
@@ -67,9 +79,8 @@ SUBSYSTEM_DEF(afk)
 	if(H.client)
 		window_flash(H.client)
 
-/datum/controller/subsystem/afk/proc/msg_admins(mob/living/carbon/human/H, mins_afk,  turf/location, action, info)
+/datum/controller/subsystem/afk/proc/log_admins(mob/living/carbon/human/H, mins_afk, turf/location, action, info)
```

Comment:
Just a nitpick but I hate how closely this proc is named to `log_admin`, and it seems ripe for confusion if anyone looks at this down the road

Reply:
Good point. Changed the name to `log_afk_action`